tests/testthat/test-derive_var_relative_flag.R

```diff
@@ -0,0 +1,38 @@
+
+## Test 1: flag observations up to first PD ----
```

Comment:
I think we should add some more tests here, e.g. to try out where mode = "last", selection = "after", inclusive = FALSE and flag_no_ref_groups = FALSE.

Reply:
Test added.